neuro_san/service/http/server/http_server.py

```diff
@@ -144,7 +144,7 @@ def make_app(self, requests_limit: int, logger: EventLoopLogger):
         handlers.append((r"/api/v1/([^/]+)/connectivity", ConnectivityHandler, request_initialize_data))
         handlers.append((r"/api/v1/([^/]+)/streaming_chat", StreamingChatHandler, request_initialize_data))
 
-        return HttpServerApp(handlers, requests_limit, logger)
+        return HttpServerApp(handlers, requests_limit, logger, self.forwarded_request_metadata)
 
```

Comment:
Pass list of metadata keys to HttpServerApp - will be used in extended request logger.

Reply:
Could this also help the 1C guys intermittent lack of request metadata in their logs?